### ISSUE TITLE
UI elements need better visual separation

### ISSUE THREAD

**Issue by danielgafni** [jenslys/zed-catppuccin-blur#6]
Hey! 

Thanks for the amazing theme! 

It's working great for me, the only problem I have is the lack of visual separation between different UI surfaces. 

Here is an example where they kinda blend together:

![image](https://github.com/user-attachments/assets/b66f44cc-e517-43af-a22b-fa925fbbb072)

**Comment by jenslys** [comment]
Il create a variant of each with higher contrast, more in line with the original theme.

**Comment by danielgafni** [comment]
Issue with Rust errors:

![Image](https://github.com/user-attachments/assets/20c2c653-ee64-4a26-a60f-052e5576920a)

**Comment by jenslys** [comment]
What zed and theme version are u on?

related: https://github.com/jenslys/zed-catppuccin-blur/issues/9

**Comment by jenslys** [comment]
https://github.com/zed-industries/extensions/pull/3328

Working on adding subtle lines to separate windows; will be tweaking them along the way.

**Comment by danielgafni** [comment]
Thanks @jenslys, that's amazing! Might try your theme again once this is merged

**Comment by jenslys** [comment]
I’ve tried various approaches to resolve this, but it appears to be a limitation or a bug with Zed. Whenever I attempt to apply a different color or change the translucency level on any of the other panels, they become completely solid. The best I can currently achieve is customizing the borders.

**Comment by Emilostuff** [comment]
Thank you for the lines. A big improvement! I wonder if it is possible in any way to make the lines change color with the background as well – or is that the issue you are talking about above? Depending on what I have running behind Zed the lines may be clearly visible (on light background) or not at all (darker background). It would be awesome they were visible in all situations.

**Comment by danielgafni** [comment]
I agree the current lines are a big improvement! I switched to the theme and will try to daily-drive it.
